next.config.js

```diff
@@ -0,0 +1,7 @@
+// Fix for esm modules segfaulting next.js
```

Comment:
@ThisIsMissEm - so I thought this shouldn't be necessary, only if we saw the SegFaults again. Or did you still see SegFaults during development of this PR, and so we definitely need this...?

Reply:
I think they're still present, I haven't tested fully without this